js/image/Example.svelte

```diff
@@ -26,10 +29,12 @@
 	.container.selected {
 		border-color: var(--border-color-accent);
 	}
+	.border.table {
+		border: 2px solid var(--border-color-primary);
+	}
 
 	.container.table {
 		margin: 0 auto;
-		border: 2px solid var(--border-color-primary);
```

Comment:
In the gallery view there's still a border being applied

<img width="1302" alt="image" src="https://github.com/gradio-app/gradio/assets/41651716/6a03f10a-3a73-4fce-b52b-a32d835ca86b">